packages/driver/src/util/serialization/index.ts

```diff
@@ -72,7 +72,7 @@ const convertObjectToSerializableLiteral = (obj): typeof obj => {
     })
 
     currentObjectRef = Object.getPrototypeOf(currentObjectRef)
-  } while (currentObjectRef)
+  } while (currentObjectRef && currentObjectRef !== Object.prototype && currentObjectRef !== Date.prototype)
```

Comment:
This is a performance optimization I noticed while debugging. It's otherwise unrelated to this PR.

If you turn on "break on exceptions" and add "break on caught exceptions" in the chromium debugger, you can see that we hit this code literally thousands of times for every test, because for every object we try to serialize, we recurse down to Object.prototype and iterate over its properties, throwing and catching dozens of "can't serialize native code" exceptions.

Same with dates - we see them as objects, recurse down into the prototype, and then throw and catch dozens of exceptions.